speechmatics/models.py

```diff
@@ -220,12 +220,12 @@ class AutoChaptersConfig:
 
 @dataclass
 class AudioEventsConfig:
-    types: Optional[List[str]]
+    types: Optional[List[str]] = None
     """Optional list of audio event types to detect."""
 
     def asdict(self):
         if self.types is None:
-            self.types = []
+            return {}
```

Comment:
do we need this if at all now? Is asdict(self) not doing the right thing already?

Reply:
asdict(self) returns {'types': None} which is not what we want. to comply with the API Spec we want {}